db/migrate/20230119181437_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+
+    add_reference :posts, :user, foreign_key: true
```

Comment:
- [ ] You can add a reference for the `author` here because the posts belong to the author
e.g
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```